test/pa11y.conf.js

```diff
@@ -0,0 +1,64 @@
+/* jshint esnext: true */
+
+const config = {
+
+  // test against WCAG 2.0 AA
+  standard: 'WCAG2AA',
+
+  // only fail on errors
+  level: 'error',
+
+  // wait 500ms before checking
+  wait: 500
+};
+
+/**
+ * Validations that we can safely ignore:
+ */
```

Comment:
@nickbristow would you be up for taking a look at these warnings we're ignoring?